File: C/Q22.c
#include <stdio.h>

int main() {
    int a, b, gcd, lcm, x, y;
    scanf("%d %d", &a, &b);
    x = a;
    y = b;

    while (y != 0) {
        int temp = y;
        y = x % y;
        x = temp;
    }
    gcd = x;

    lcm = (a * b) / gcd;
    printf("LCM: %d", lcm);
    return 0;
}

File: Python/Q22.py
# Q22.py - Program to Find LCM of Two Numbers
def lcm(x, y):
    from math import gcd
    return abs(x*y)//gcd(x, y)

num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))
print(f"LCM is {lcm(num1, num2)}")